devicetypes/krlaframboise/sensative-strips-drip.src/sensative-strips-drip.groovy

```diff
@@ -0,0 +1,776 @@
+/*
```

Comment:
Since **zwave-water-temp-light-sensor.groovy** already supports this device - if there are any changes needed, please add them in this DTH:
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zwave-water-temp-light-sensor.src/zwave-water-temp-light-sensor.groovy

Reply:
Hello Konrad,

We had received the following feedback from SmartThings in the past,

"We are in the middle of certifying Comfort and Drip and ran into an unusual situation.  It seems that the devices share a Z-Wave certification; meaning that when we pair the devices, we have no way of differentiating between the two products.  Since they share the same code and capabilities, they share the same Device Type Handler.  Therefore, the SmartThings platform see's them as the same device."

Also, based on the valuable feedback we have received from the SmartThings end users in the past couple of years, and after consulting with SmartThings, we decided to outsource the development of SmartThings DTH that could differentiate the two products Strips Comfort and Strips Drip and also maximize the user experience of these products towards the SmartThings ecosystem. Our test team has verified these device handlers thoroughly.

Therefore, we request you to kindly review these new device handlers and adapt the same. Let us know if we need to update any changes within these new device handlers.

Best Regards,
Dhiraj Paryani

Reply:
@dparyani 
Hello Dhiraj, 

As I mentioned before, both devices are recognized (on a protocol level) as ->  mfr: "019A", prod: "0003", model: "000A" 
In other words - both models report the same values,  that's why they have the same fingerprint in a SmartThings DTH (that was mentioned above).

If the same fingerprint was added in many DTHs - it wouldn't work, it wouldn't be deterministic what DTH would be matched at the moment of pairing the device.

To differentiate the two products:  Strips Comfort and Strips Drip they would have to report different **model** first.
(that way we would let two separate fingerprints in the zwave-water-temp-light-sensor.groovy DTH).

What I would suggest here to create a new pull request (instead of #50493 and #50494) and change the fingerprint to:
**fingerprint mfr: "019A", prod: "0003", model: "000A", deviceJoinName: "Sensative Multipurpose Sensor" //Sensative Strips Comfort/Drip** in this DTH:
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zwave-water-temp-light-sensor.src/zwave-water-temp-light-sensor.groovy

That way it would be less confusing for users,  who - by the way - can change the displayed name of the device right after pairing the device (but they can do it any time in the application).